### DISCUSSION TITLE
Why i am not able to send large file example 200MB or 500MB from client to server only 16KB is getting stored in server using quic?

### DISCUSSION THREAD

**Discussion by Prathi1729** [aiortc/aioquic#408] [Q&A]
<EOM>

**Comment by devkdas** [comment]
@Prathi1729 try these info from here
https://github.com/Karthikdasari0423/Download-files-using-AIOQUIC

**Comment by Laevon** [comment]
When file is slightly larger, aioquic will slice the content and send them over one stream sequentially. When streamdatareceived event triggers, save and append the data if stream_id is the same. That's what I found. Have you ever solved this problem? I'm confused that large file took too much time to transfer, even in local network, ~70 sec to transfer one ~80 MB file in local network without adding delay or loss rate.
